checkout.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>>Adopt A Dog</title>
+
+  <link rel="stylesheet" type="text/css" href="./styles/checkout.css">
+
+</head>
+
+<body>
+  <header>
+    <img src="./images/adoptadog-logo.svg" alt="adopt a dog logo">
+    <h1>Seattle's Premiere dog adoption agency</h1>
+    <img src="./images/cart-icon.svg" alt="shopping cart icon">
+    <p>check out</p>
+  </header>
+
+  <main>
+    <nav class="col span_3_of_12">
+      <ul class="menunavigation">
+        <li class="menunavigationitem"><a href="./index.html">Home</a></li>
+        <li class="menunavigationitem"><a href="./doglist.html">Dogs</a></li>
+        <li class="menunavigationitem"><a href="./blog.html">Blog</a></li>
+      </ul>
+    </nav>
+    <h2>Checkout</h2>
+    <form class="" method="get" name="check-out-cart">
+      <div class="row">
+        <label for="name" Autofocus>Name
+          <input type="text" name="name" value="name" id="name" required>
+        </label>
+      </div>
+
+      <div class="row">
+        <label for="email">Email Address
+          <input type="email" name="email" value="email" id="email" required>
+        </label>
+      </div>
+
+      <div class="row">
+        <label for="address">Street Address
+          <input type="text" name="address" value="address" id="address" required>
+        </label>
+      </div>
+
+      <div class="row">
+        <label for="city">City
+          <input type="text" name="city" value="city" id="city" required>
+        </label>
+
+        <label for="state">State
+          <input type="text" name="state" value="state" id="state" required>
+        </label>
+
+        <label for="zipcode">Zip Code
+          <input type="number"
```

Comment:
Nice guess at using number for a zip code, but zip codes (and phone numbers, etc) are generally collected as strings.